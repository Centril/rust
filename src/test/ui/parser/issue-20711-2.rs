struct Foo;

impl Foo {
    fn foo() {}

    #[stable(feature = "rust1", since = "1.0.0")]
} //~ ERROR expected one of `async`, `const`, `crate`,

fn main() {}
