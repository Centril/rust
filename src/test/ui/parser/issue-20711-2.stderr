error: expected one of `async`, `const`, `crate`, `default`, `extern`, `fn`, `pub`, `super`, `type`, or `unsafe`, found `}`
  --> $DIR/issue-20711-2.rs:7:1
   |
LL |     #[stable(feature = "rust1", since = "1.0.0")]
   |                                                  - expected one of 10 possible tokens
LL | }
   | ^ unexpected token

error: aborting due to previous error

